25/4/14
========================================================

### Last couple of weeks

- Made code for reproduction methods plot, haven't got it to work properly yet
- Unit tested all lib files in fragmented_genome_with_snps, made changes to unit tests to reflect lib file changes
- Learned how to make gems
- Made the pdist and pmeth gems
- Installed all neccesary software/gems on the hpc
- Ran the small_dataset2 (run11) in the cluster, it ran but figures could not be generated, due to lack of x11. Might be more feasible to generate distribution plots on desktop after the algorithm has run.

### Next

1. Run the genetic algorithm on small_dataset2 in cluster and time it, for comparison with desktop

7. Run the algorithm on a full Arabidopsis chromosome 4 model in the cluster
 - May need to parallelize the algorithm: perhaps use the parallel gem
 
5. Minor things:
 - Replace extra randoms with mutants?

2. Update gg-plots to be single figures, with all metrics, but faceted, on same scale if possible (different colours for different metrics)
 - Use this to evaluate results

7. **Find a suitable distribution for real EMS backcross experiment SNPs**

8. Write code that can separate homozygous and heterozygous SNPs from a VCF file from real data: add this to ReformRatio

9. (not essential) - fix code for making plot of which recombination method works best

### Eventually

6. Understand maths behind peak finding, and use peak finding in R: add to GA

7. Update README further with the "Purpose of Project" and "Peak Finding" sections

2. Use rdoc, and do documentation for key ruby files.

3. Make ruby **gems** for the project: GATOC and permutation metrics and a general genetic algorithm

