2/4/14
========================================================

### Last week

- Created a smaller test dataset to evaluate whether the genetic algorithm will work
- Running the algorithm on small test dataset

### Next

1. Make performance figures for small dataset run so far

2. Update gg-plots to be single figures, with all metrics, but faceted, on same scale if possible (different colours for different metrics)

1. Finish running the GA for small dataset, comparing against the same SNPs used to make the model genome (example ratio in Q-Q plot)

2. Run the GA on small dataset again, with the same distributions but not exactly the same SNPs (as example ratio in Q-Q plot)

3. Write up results of running the algorithm with a small dataset

3. Write a method that works out the genome length, put into ReformRatio

4. Update unit tests for GATOC, and annotations for SNP_dist

5. Minor things:
 - Replace extra randoms with mutants?
 - Make z a customizable paramter in evolve in GATOC
 - Record average fitness for the correct permutation for each dataset
 
6. Write code that will determine which method in GATOC is contributing most significantly to the improvements

7. Merge the test branch with master, then update README with correct info on how to run everything

7. Run the algorithm on a full Arabidopsis chromosome 4 model in the cluster
 - May need to parallelize the algorithm: perhaps use the parallel gem

7. Find a suitable distribution for real EMS backcross experiment SNPs

8. Write code that can separate homozygous and heterozygous SNPs from a VCF file from real data: add this to ReformRatio 

### Eventually

6. Understand maths behind peak finding, and use peak finding in R: add to GA

7. Update README further with the "Purpose of Project" and "Peak Finding" sections

2. Use rdoc, and do documentation for key ruby files.

3. Make ruby **gems** for the project: GATOC and permutation metrics and a general genetic algorithm

