4/4/14
========================================================

### Today

- Unit tested lib files
- Added SNP lists to output of Genetic algorithm

### Next

6. Write code that will determine which method in GATOC is contributing most significantly to the improvements

7. Write code that will compare the genomic positions of SNPs from permutations, with the actual positions in model (hm, ht and hyp)
 - For the small dataset: If the positions are similar, when the contigs are ordered differently, this could support my idea that the postions end up being similar because of the simplicity of the model
 - For the large dataset: If the positions are similar, when the contigs are ordered differently, then this is a big problem, as it will mean the ratio distribution can't be used to reorder contigs
 - However, assuming this ^ isn't the case, then running the algorithm should get us to the only permutation that reforms the ratio (the correct one), but only if it can avoid local maxima
 - Need to write code that will get the SNP lists for the original order contigs, including an example of hyp
 - We have the SNP lists being outputted, so we can potentially move this to after/during the below things

7. Merge the test branch with master, then update README with correct info on how to run everything

7. Run the algorithm on a full Arabidopsis chromosome 4 model in the cluster
 - May need to parallelize the algorithm: perhaps use the parallel gem
 
5. Minor things:
 - Replace extra randoms with mutants?

2. Update gg-plots to be single figures, with all metrics, but faceted, on same scale if possible (different colours for different metrics)
 - Use this to evaluate results

7. Find a suitable distribution for real EMS backcross experiment SNPs

8. Write code that can separate homozygous and heterozygous SNPs from a VCF file from real data: add this to ReformRatio 

### Eventually

6. Understand maths behind peak finding, and use peak finding in R: add to GA

7. Update README further with the "Purpose of Project" and "Peak Finding" sections

2. Use rdoc, and do documentation for key ruby files.

3. Make ruby **gems** for the project: GATOC and permutation metrics and a general genetic algorithm

