09/10/13
========================================================

fragmented_genome_with_snps repo:
---------------------------------

>### commit 
>- rearrangement methods setup
>https://github.com/edwardchalstrey1/fragmented_genome_with_snps/commit/d77016e13e1dc161b11c8bb8c4c0b7337e0b36cd
>- density and positions to json
>-https://github.com/edwardchalstrey1/fragmented_genome_with_snps/commit/7e5fa2271e4425f68c09c3eba4cf58fedea318b9

>## Today
>1. Put all of the code in json->fasta.rb and density_method_testa.rb into coherent ruby methods

>2. Started a new working file called rearrangement_methods.rb 

>## Next 
>1. Put the 'score determining algorithm' into a ruby method that can be easily called when testing rearrangement methods 

>2. Re-write the fragment rearrangement methods already tried into coherent ruby methods, testing each one as you go and writing results

>3. Come up with new methods for rearranging the fragments by snp density, then test them

>## Also you could...
>1. Put the methods from each working file into classes?

>2. Write an extra method in fragments_w_snps.rb that tests whether the snp positions are really in a normal distribution
