10/10/13
========================================================

fragmented_genome_with_snps repo:
---------------------------------

>### commit
>- see commits from 10/10/13
>https://github.com/edwardchalstrey1/fragmented_genome_with_snps/commits/master

>## Today
>1. Finished putting all of the code in working files into coherent methods

>2. Started rearrangement_methods.md which is a summary of the experiments being done in rearrangements.rb

>## Next
>1. Possibly: Do better versions of the left/right method:
>  - Run the existing left/right method on the density ordered frags (can just re-order lengths and positions using the ids as hash keys), then reverse the right before adding it to left (and flattening)
>  - Run the l/r method on fasta order (as it is currently), then go back and order the left and right lists by density (individually)
>  - This may not be worth it though, based on the current result of the l/r method. The second of the above will be particularly time consuming.

>2. Come up with other new methods for rearranging the fragments by snp density, then test them

>3. Make histogram

>## Also you could...
>1. Put the methods from each working file into classes?

>2. Write an extra method in fragments_w_snps.rb that tests whether the snp positions are really in a normal distribution
