11/11/13
========================================================

Today
----

- Updated model genome to [Arabidopsis chromosome 4](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/writeup/arabidopsis_chromosome4.md) - see here for what I did.

- See commits from 11/11/13 in [fragmented genome repo](https://github.com/edwardchalstrey1/fragmented_genome_with_snps)

Next
----

0. Update [README](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/README.md)/ write a new one:
 - summarize the aims of the mini project more exactly with what files do/ what to read in what order

1. Update density_method_testa.rb to work with new datasets (e.g. ARGV) and run for each dataset

2. Create more datasets

3. Update rearrangement_methods.rb to work with new datasets (e.g. ARGV) and run for each dataset, saving figure creation txts in appropriate places
 - actually... it might be better to simplify the png/figure creation process massively using rinruby... or not, you may not want to create figures for each dataset???
 - maybe put all those id txts creation into a method?
 - same goes for skew_scatta.R/json->fasta.rb

4. Need to create the super scatter graph to test whether skew mwthod is effective: update/make work [skew_scatta.R](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/skew_scatta.R)
 - make a version with each sub graph as part of the main?

5. Write up rearrangement methods in a new .md for the **average scores** of multiple datasets for each method

6. Write up your findings from the super scatter graph

Later on
-----

1. Make an improved skew method, if it looks like the skew is a good idea

2. Make a linear programming rearrangement method

3. Way down the line: test different midpoints for your rearranged distribution to be around; this could be after you already have a rearrangement method that works when we know the midpoint is in the middle. Write a method that uses that rearrangement method multiple times, with different midpoints, to find the best one, and therefore estimate the mutant position.