12/11/13
========================================================

Fragmented Genome:
---------

### Today

- Updated fragmented genome [README](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/README.md), to be more informative

- Updated density_method_testa.rb and rearrangement_methods.rb to work with the arabidopsis datasets, see commits from 12/11/13 [fragmented genome repo](https://github.com/edwardchalstrey1/fragmented_genome_with_snps)

- Started updating [skew_scatta.R](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/skew_scatta.R) so that it will generate graphs of the gradients for fragments with 6 or more SNPs, for the arab datasets

### Next

4. Continue the super scatter graph to test whether skew method is effective: update/make work [skew_scatta.R](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/skew_scatta.R)
 - make a version with each sub graph as part of the main?
 - sort out bandwidth issue

5. Write up rearrangement methods in a new .md for the **average scores** of multiple datasets for each method

6. Write up your findings from the super scatter graph

### Later on

1. Make an improved skew method, if it looks like the skew is a good idea

2. Make a linear programming rearrangement method

3. Way down the line: test different midpoints for your rearranged distribution to be around; this could be after you already have a rearrangement method that works when we know the midpoint is in the middle. Write a method that uses that rearrangement method multiple times, with different midpoints, to find the best one, and therefore estimate the mutant position.

4. Eventually you will need to modify density_method_testa.rb to work with any ARGV file, and of course the file containing the rearrangement/mutation locating algorithm, saving the files in an appropriate location

Literature Review
-------

- Read papers on the topics Dan has suggested

- Write appropriate sections, focusing on genetics as much as bioinformatics