13/11/13
========================================================

Fragmented Genome:
---------

### Today

- Finished [skew_scatta.R](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/skew_scatta.R) and for dataset 3, and made scripts to generate other figures: [ruby](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/scatter_graphs.rb) [R](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/scatter_vectors.R) 

- Generated [figures](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/tree/master/arabidopsis_datasets/dataset3/figures)

### Next

1. Make another super scatter that uses kernel density estimate for entire original order: where n makes the sections of the graph equivalent to the sub graphs. Or a plot of multiple graphs somehow.

2. Sort out random_vector_Rcode.txt for dataset 3/ or all datasets, so you can include barplot of scores in writeup

3. Write up plots for dataset 3, comparing d3 and d2 original order plots

4. Need to make it clear in writeup that for the super scatter, the fragments on axis in original order, but excluding fragments < 6 SNPs

5. Write up rearrangement methods in a new .md for the **average scores** of multiple datasets for each method

6. Write up your findings from the super scatter graph

### Later on

1. Make an improved skew method, if it looks like the skew is a good idea

2. Make a linear programming rearrangement method

3. Way down the line: test different midpoints for your rearranged distribution to be around; this could be after you already have a rearrangement method that works when we know the midpoint is in the middle. Write a method that uses that rearrangement method multiple times, with different midpoints, to find the best one, and therefore estimate the mutant position.

4. Eventually you will need to modify density_method_testa.rb to work with any ARGV file, and of course the file containing the rearrangement/mutation locating algorithm, saving the files in an appropriate location

Literature Review
-------

- Read papers on the topics Dan has suggested

- Write appropriate sections, focusing on genetics as much as bioinformatics
