14/11/13
========================================================

Fragmented Genome:
---------

### Today

1. Written up scores for datasets 3-6 in a table

2. Written up plots of original order vs SNP density for datasets 2/6

3. Started writing up the skew scatter plots

### Next

1. Write up your findings from the super scatter graph
 - Need to make it clear in writeup that for the super scatter, the fragments on axis in original order, but excluding fragments < 6 SNPs
 - Compare figures of K density estimate to SNPs per Kb (d_o)
 - Remember the example gradient graphs are for frags with 6+ SNPS only. could this be a problem?
 - [super scatter](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/arabidopsis_datasets/dataset6/figures/genome_kdens_snps.png) is all SNPs on genome, but only goes up to highest SNP

### Later on

1. Make an improved skew method, if it looks like the skew is a good idea
 - This could involve **eliminating all fragments that are definitely not from the centre of the distribution** by their skew... if possible

2. Make a linear programming rearrangement method

3. Test different midpoints for your rearranged distribution to be around; this could be after you already have a rearrangement method that works when we know the midpoint is in the middle. Write a method that uses that rearrangement method multiple times, with different midpoints, to find the best one, and therefore estimate the mutant position.

4. Eventually you will need to modify density_method_testa.rb to work with any ARGV file, and of course the file containing the rearrangement/mutation locating algorithm, saving the files in an appropriate location

Literature Review
-------

- Read papers on the topics Dan has suggested

- Write appropriate sections, focusing on genetics as much as bioinformatics
