6/11/13
========================================================

Today
-----

- Continued literature review

- Added references bib file

Next
----

- You need to re-focus on and devote a major section to **using an alignment based method on a draft genome and then working the order of the contigs**. Maybe update the plan first.

- Lit review: remember to include examples of software

- Read more relevant papers on de novo SNP calling?

Later
-----

- Finish [skew_scatta.R](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/skew_scatta.R), it uses Kernel Density estimate, but it doesn't work yet, see the annotations

- Include R code (or links to script preferably) in [rearrangement methods](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/writeup/rearrangement_methods.md) and [part2](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/writeup/p2_rearrangement_methods.md), you will need to look at older commits of [skew_scatta.R](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/skew_scatta.R) before 4/11/13. Also check that the other R scripts have not changed since they generated graphs

- see **Fragment Rearrangement** in [31st.md](https://github.com/edwardchalstrey1/tsl_lab-book/blob/master/October_13/31:10:13.md)

