09/12/13
========================================================

**From now on, each day after tea break do some work on literature review OR read calculus/statistics R**

### Today

- Discovered that the crossing experiment I am doing doesn't always give a normal distribution, see Ashelford et al 2011

- wrote a doc to help understand the [difference between my crossing experiment and the one in the NGM paper](https://github.com/edwardchalstrey1/tsl_lab-book/blob/master/project_reminders.md)

- Started thinking about hill climbing and machine learning

### Next

3. Make the model genome accurate: back-crossed EMS mutant
 - need to evaluate, by looking at multiple sources, what the SNP distribution is like in back-crossed EMS mutants
 - is it a normal distribution?
 - what is the shape of the curve/ standard deviation?
 - contig/fragment sizes: we can test different sizes, actually we should do this, sizes are down to experimental choice.
 - then obviously modify the model genome as appropriate

2. Research into machine learning and hill climbing algorithms, as an alternative for fragment rearrangement

1. Try and work out why exactly the centre of the normal dist is not where the mutation is
 - This may not be a real problem, have generated datasets of equal frag size to check, still seems to be the case
 - We can sort this out, if it is still a problem, after modifying the model genome

2. Read other papers for the SNP calling mapping based/background section, lit review

3. Possibly: use rdoc, and do documentation for key ruby files, but need to classify them first

### Later on

2. Make a better rearrangement method: linear programming?

3. Test different midpoints for your rearranged distribution to be around; this could be after you already have a rearrangement method that works when we know the midpoint is in the middle. Write a method that uses that rearrangement method multiple times, with different midpoints, to find the best one, and therefore estimate the mutant position.

4. Eventually you will need to modify density_method_testa.rb to work with any ARGV file, and of course the file containing the rearrangement/mutation locating algorithm, saving the files in an appropriate location

