13/12/13
========================================================

**From now on, each day after tea break do some work on literature review OR read calculus/statistics R**

### Today

- Perfected the [ratio distribution](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/ratio/ratio.R) (I think)

- Added a [new branch](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/tree/ratio) on github

### Next

3. Make the model genome accurate: back-crossed EMS mutant
 - Modify arabidopsis_w... to have the new SNPs
 - Need a way of distinguishing between heterozygous and homozygous SNPs
 - Then a new fragment rearrangement method will be required: reform the ratio by puttting frags in original order, then try with different orders
 - see [what I need to do for the model genome](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/writeup/ideas.md)
 - contig/fragment sizes: we can test different sizes, actually we should do this, sizes are down to experimental choice.

2. Research into machine learning and hill climbing algorithms, as an alternative for fragment rearrangement

3. After you've done at least some testing of hill climbing rearrangement algorithms, re-write the readme

2. Read other papers for the SNP calling mapping based/background section, lit review

### Possibly

3. Possibly: use rdoc, and do documentation for key ruby files, but need to classify them first

1. Try and work out why exactly the centre of the normal dist is not where the mutation is
 - This may not be a real problem, have generated datasets of equal frag size to check, still seems to be the case
 - We can sort this out, if it is still a problem, after modifying the model genome

### Later on

2. Make a better rearrangement method: linear programming?

3. Test different midpoints for your rearranged distribution to be around; this could be after you already have a rearrangement method that works when we know the midpoint is in the middle. Write a method that uses that rearrangement method multiple times, with different midpoints, to find the best one, and therefore estimate the mutant position.

4. Eventually you will need to modify density_method_testa.rb to work with any ARGV file, and of course the file containing the rearrangement/mutation locating algorithm, saving the files in an appropriate location

