18/12/13
========================================================

### Today

- Modified arabidopsis_c4...rb to have the homozygous/hetero SNPs from ratio.R
- Thought about brute force algorithm, updated ideas.md


### Main

0. Plan for lab meeting

1. Update basecamp

3. Make the model genome accurate: back-crossed EMS mutant, see [ideas](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/writeup/ideas.md)
 - Just need to find out why one SNP is replicated???
 - Just use one dataset for now, to test rearrangement methods
 
4. Testing the normal distribution of hom/het SNP ratio
 - write code to reform the ratio from FASTA/VCF files, by puttting frags in original order, by using a qq plot to get R^2 value
 - contig/fragment sizes: we can test different sizes, sizes are down to experimental choice.
 
6. Evaluate feasibility of brute force, to reorder fragments to get the correct R^2 value
 
5. Rearrangement: read [ideas](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/writeup/ideas.md), and update continuously

3. After you've done at least some testing of new rearrangement algorithms, for the ratio re-write the readme, with detailed intructions on using files e.g. command line options

### Other

2. Read other papers for the SNP calling mapping based/background section, lit review

3. Read up on calculus/statistics in R

### Possibly

3. Possibly: use rdoc, and do documentation for key ruby files, but need to classify them first

3. For a succesful rearrangement method that only recognises a normal distribution in the middle of the genome: Test different midpoints for your rearranged distribution to be around; this could be after you already have a rearrangement method that works when we know the midpoint is in the middle. Write a method that uses that rearrangement method multiple times, with different midpoints, to find the best one, and therefore estimate the mutant position.

