20/5/14
========================================================

### This week

- SNP distribution plots can now be made after the algorithm has run with [distribution_plots.rb](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/distribution_plots.rb)
- Type and fitness scores of permutations in each generation can now be viewed in command line with [view_table_data.rb](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/view_table_data.rb)

### Next

1. Add algorithm quit condition: linear regression of the best fitness from last 5 generations, gradient below a certain threshold

2. Ask Martin to help with sorting out reason for R memory usage problem - looking at current cluster runs

3. Add near swap method to PMeth - adjacent objects swap position. Add this into genetic algorithm

4. Run some more instances of the algorithm in the cluster

5. Change all unit tests to use setup method with instance variables

7. **Find a suitable distribution for real EMS backcross experiment SNPs**

8. Write code that can separate homozygous and heterozygous SNPs from a VCF file from real data: add this to ReformRatio

### Extra analysis

3. (not essential) - fix code for making plot of which reproduction method works best, test on small_dataset2 results (use unit tests)

2. Update gg-plots to be single figures, with all metrics, but faceted, on same scale if possible (different colours for different metrics)

### Eventually

1. Understand maths behind peak finding, and use peak finding in R: add to GA

2. Update README further with the "Purpose of Project" and "Peak Finding" sections

3. Use rdoc, and do documentation for key ruby files.

