9/5/14
========================================================

### This week

- Added rake files to project and gems
- Started one run of the algorithm on 10K_dataset3 in cluster Friday 2nd
- Unit tested and pdist and pmeth some more
- Updated project to use pmeth gem
- Spent a long time fixing issue with new_population method in GATOC, because using pmeth gem
- Wrote script for running algorithm many times in cluster: ran it Friday 9th

### Next

1. Investigate saving permutations in some kind of table for easy access, take input as the txt files output from algorithm for now

2. Get travis working with pdist and pmeth AND fragmented_genome_with_snps

3. (not essential) - fix code for making plot of which reproduction method works best, test on small_dataset2 results

4. Check if cluster making pngs - since it probably isn't, make a script that can do it from the permutation lists (txts)

2. Update gg-plots to be single figures, with all metrics, but faceted, on same scale if possible (different colours for different metrics)
 - use pdist gem

7. **Find a suitable distribution for real EMS backcross experiment SNPs**

8. Write code that can separate homozygous and heterozygous SNPs from a VCF file from real data: add this to ReformRatio

### Eventually

1. Understand maths behind peak finding, and use peak finding in R: add to GA

2. Update README further with the "Purpose of Project" and "Peak Finding" sections

3. Use rdoc, and do documentation for key ruby files.

