08/10/13
========================================================

fragmented_genome_with_snps repo:
---------------------------------

>### commit
>- tidy methods and one json
>https://github.com/edwardchalstrey1/fragmented_genome_with_snps/commit/0a9637d8bad61e8be4f33d8998e53ee6fe243f7d

>## Today
>1. Neatened up the methods in json->fasta.rb and method_testa2.rb (used it to overwrite fragments_w_snps.rb). fragments_w_snps.rb now produces just one json file that can be read into json->fasta.rb, which in turn produces a fasta and a vcf file.

>2. Fixed the problem I was having with the pos_each_frag method (fragments_w_snps.rb), which is supposed to work out the position of snps on each fragment, using the position information for the whole sequence. It now does this in the same way as older commits of fragments_w_snps.rb. I was trying to work out a more elegant way of doing this (see greyed out parts of the get_frags method), but couldn't get anything substantially simpler to work. I may come back to this at a later date, but it is not of immediate concern.

>## Next
>1. Neaten up the code for determining the snp density of each frag in find_mutant_pos_snp_density_frags.rb - put into ruby methods

>2. Put the 'score determining algorithm' into a ruby method that can be easily called when testing rearrangement methods 

>3. Re-write the fragment rearrangement methods already tried into coherent ruby methods, testing each one as you go and writing results

>4. Come up with new methods for rearranging the fragments by snp density, then test them

>## Also you could...
>1. Put the methods from each working file into classes?

>2. Write an extra method in fragments_w_snps.rb that tests whether the snp positions are really in a normal distribution
