25/10/13
========================================================

repo: [fragmented_genome](https://github.com/edwardchalstrey1/fragmented_genome_with_snps)

see commits from 25/10/13

Today
-------

- Made the corrections Dan suggested for rearrangement_methods.md

- Made ordinal similarity.png better

- Improved the normal distribution of SNPs in the model genome, fragments_w_snps.rb. The skew based rearrangement methods should work better now.

- Made README more efficient

Next
----

### Fragment rearragement

2. Improving the model genome
 - You have the fasta/vcf used for the current methods saved, make new fasta/vcf
 - **You may want to update fragments_w_snps.rb even further: At the moment the breaks are set to 4000 (which means within each 50b the positions are still random for 200kb seq.), the fragments are still 50-250b long, perhaps it would be better if they were longer to make sure there is a skew. Also, longer fragments will mean less frags with zero snps.**
 - Writeup changes/differences in new markdown
 - Run existing rearrangement methods on new fasta/vcf
 - Writeup the results in new markdown
 
3. Improve the left/right method further (somehow, if possible) rename **Skew Method**
 - Run it on the improved model genome
 - Writeup in the new markdown
 - Create a scatter graph of the gradient (for each frag) against the positions/ frags in rearranged order (for the existing skew method first!)

4. Try to do a linear programming method

5. Way down the line: test different midpoints for your rearranged distribution to be around; this could be after you already have a rearrangement method that works when we know the midpoint is in the middle. Write a method that uses that rearrangement method multiple times, with different midpoints, to find the best one, and therefore estimate the mutant position.

### [Literature review repo](https://github.com/edwardchalstrey1/lit_review_snp_detection_calling)

- Research any background for the work that I will eventually be doing, so that the person reading the review could understand my work after

- Carry on reading legget et al

