28/10/13
========================================================

repo: [fragmented_genome](https://github.com/edwardchalstrey1/fragmented_genome_with_snps)

see commits from 28/10/13

Today
-------

- Created a new model genome dataset (dataset 2)

- Written up the need for creating the new model genome

- Ran existing rearrangement methods on new fasta/vcf

- Started writeup of the results in a new markdown

Next
----

### Fragment rearragement
 
3. Improve the left/right method further (somehow, if possible) rename **Skew Method**
 - Run it on the improved model genome
 - Writeup in the new markdown
 - **Create a scatter graph of the gradient (for each frag) against the positions/ frags in rearranged order (for the existing skew method first!)**

4. Try to do a linear programming method

5. Way down the line: test different midpoints for your rearranged distribution to be around; this could be after you already have a rearrangement method that works when we know the midpoint is in the middle. Write a method that uses that rearrangement method multiple times, with different midpoints, to find the best one, and therefore estimate the mutant position.

### [Literature review repo](https://github.com/edwardchalstrey1/lit_review_snp_detection_calling)

- Research any background for the work that I will eventually be doing, so that the person reading the review could understand my work after

- Carry on reading legget et al

