30/10/13
========================================================

repo: [fragmented_genome](https://github.com/edwardchalstrey1/fragmented_genome_with_snps)

see commit: gradient graph pre-requisites

Today
-------

- Got lists of zero's and one's for each fragment, where each 1 is a snp, each 0 a non-snp nucleotide. from [json->fasta.rb](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/json-%3Efasta.rb)

- Found out how to get the gradient of snps for each fragment in R, when x = 1:(frag length) and y = 0:1 and each point is zero or one, representing each nucleotide in the fragment. See [how to get the gradient on stack](http://stackoverflow.com/questions/19661766/how-to-get-gradient-of-abline-in-r)

Next
----

Thursday: fragment rearrangement

Friday: Lit review

### Fragment rearragement

1. Run [json->fasta.rb](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/json-%3Efasta.rb) with the JSON from dataset 2 and add it/any other important dataset 2 files to the [d2 folder](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/tree/master/fasta_vcf_d2)
 - remember you also need to create a new folder called skew_scatter beforehand

2. Write R script for x:fragment, y:snp gradient plot:
 - try and make a loop that runs the stack method to get gradient for each fragment; look up how to do loops in R
 - ideally in same script as the gradient loop: make a vector of the gradients and plot against vector of numbers (fragments 1..n)
 - make script include title/labels for the graph
 - also output one example gradient plot of a frag with labels, for showing working

3. Observe the graph and decide what to do next:
 - write up the making/reason for the above in [rearrangement_p2](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/writeup/p2_rearrangement_methods.md)
 - comment on the two plots produced: write about how 2 peaks either side of the mean expected because flat gradient for middle frags with many snps
 - have a think, based on the shape of the graph, how the left_right/skew method could potentially be improved: we assume that with real data, if the snps are normally distributed about the mutant position, then the gradients of frags would show a similar pattern: of course you may find that the fragment's are really not having noticeable skew's/gradients
 - if the gradients are really random, perhaps create a 3rd dataset where the difference between break range and fragment size is even smaller: perhaps you could research the sizes of fragments used in snp calling experiments and the number of snps they would typically have, because higher number of snps means a more reliable gradient, **each of your frags has a low number of snps because the frags are very small (read size not frag size)**

4. Create a new skew method, if you have decided this is a good idea

5. Try a linear programming based rearrangement method

6. Way down the line: test different midpoints for your rearranged distribution to be around; this could be after you already have a rearrangement method that works when we know the midpoint is in the middle. Write a method that uses that rearrangement method multiple times, with different midpoints, to find the best one, and therefore estimate the mutant position.

### [Literature review repo](https://github.com/edwardchalstrey1/lit_review_snp_detection_calling)

- Research any background for the work that I will eventually be doing, so that the person reading the review could understand my work after

- Carry on reading legget et al
