10/3/14
========================================================

### Last week

- Worked out 7 different metric scores for testing the contig rearrangement
- [Tested these metric scores for simple permutations](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/Comparing%20Permutations/comparing_permutations.md)
- Created [scripts](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/tree/master/score_plots) to generate figure(s) of the metric scores for the permutations generated by the genetic algorithm

### Next

1. Get figures for the various metrics for the algorithm that has run until quitting.

2. Add the reference of the permutation metrics paper to the [writeup](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/Comparing%20Permutations/comparing_permutations.md); and equations?

2. Make the model genome have uniform heterozygous SNPs: use real SNP data?

1. Work on a new version of the fitness method, that can place the peak in the distribution in the right place
 - use the [Kolmonov-Smirnov](http://stat.ethz.ch/R-manual/R-patched/library/stats/html/ks.test.html) test for the Q-Q plot instead of spearman?
 
2. Run the algorithm again with new dataset and fitness method

2. Parallelize the algorithm: perhaps use the parallel gem
 - **No need to worry about this until after you have the fitness method that really works!**
 - This will probably only be neccesary if you want to run it in the cluster


### Eventually

6. Understand maths behind peak finding, and use peak finding in R: add to GA

7. Update README further with the "Purpose of Project" and "Peak Finding" sections

2. Use rdoc, and do documentation for key ruby files.

3. Make ruby **gems** for the project: GATOC and permutation metrics

