23/07/14
========================================================

### Recently

- Testing new fitness methods in cluster
- Documenting project
- Can now run algoritm with any fitness method

### The final push

1. Unit test hyp_distance fitness method

2. Write up how to run entire experiment, with file names etc so reproducible
 - Comment on GATOC.evolve more
 - Write new fitness method summaries (including count/ratio) with links to README. Take from here [alt](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/Progress/Ratio_over_generations/alt_fitness.md). Add links in alt.
 - Make results directories with links to README
 - Write up how to use all analysis scripts (including circos - test this)
 - In Documentation: add descriptions for new_pop and select flow charts, and links to how fitness methods work
 - Make [do_metrics_work.md](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/Progress/Do_metrics_work/do_metrics_work.md) neater and link into README
 - Tidy away any of the Progress files that are completely irrelevant

1. Run algorithm on small_dataset with max_hyp and hyp_distance
 - do_metrics_work for hyp_distance

3. Modify the gifs, then add into all the current results:
 - Background of correct permutation, with moving permutations on top
 - Both fitness methods in different colours for comparison?

4. Results
 - Make Results with small datasets for each fitness method: gifs
 - Analyse the results of what is running in the cluster currently, with the existing analysis scripts: if cluster runs not finished do anyway, can redo this at end
 - Explain the problem with max_density
 - Explain that max_density and snp_distance both use homozyogus SNPs only, and this would not be useful for real data as the peak is in the ratio
 - Do circos for examples of each version of the algorithm (fitness method)
 
5. Peak finding
 - Use count and ratio to find the **highest ratio** for finding the causative SNP, and conclude this div as the region of interest OR find a way to get the same peak/candidate
 - Look at the contig(s) in this region/ that have the causative in the correctly ordered genome. Where are this/these contigs in permutations.
 
6. Final investigation / future ideas writeup
 - Solving the problem with max_density: max_ratio may become viable
 - Another idea: max value of ratio with div=10K (or appropriate). Write the code and run do_metrics_work if time.
 - More realistic model genome with low number of homozygous SNPs?
 - Mention that code is needed to separate homozygous and heterozygous SNPs from a VCF file from real data

