20/1/14
========================================================

### Today

- Added variation (ran argument) into the new_population method of [genetic algorithm for fragment rearrangement](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/ratio/reform_ratio.rb)

- Started trying to get the theoretical normal distribution (qqnorm) to yield a useful correlation value

- **ruby reform_ratio_old dataset_name shuffled** will give you text files, sourcing ratio_old.R shows the qq plot

### Main

2. Make sure the GA works for a variety of distributions: 
 - Test the GA against different distributions in the qq plot, e.g. a **theoretical distribution**
 - contig/fragment sizes: we can test different sizes, sizes are down to experimental choice.
 
1. Work out a way of determining the causative from the distribution: use calculus to find the top of the peak
 
6. Evaluate feasibility of brute force, to reorder fragments to get the correct coefficient value
 - I don't think this is feasible
 
5. Write up what you've done since [ideas](https://github.com/edwardchalstrey1/fragmented_genome_with_snps/blob/master/writeup/ideas.md), and update continuously

### Other

2. Read other papers for the SNP calling mapping based/background section, lit review

3. Read up on calculus/statistics in R

### Possibly

3. Possibly: use rdoc, and do documentation for key ruby files, but need to classify them first

3. For a succesful rearrangement method that only recognises a normal distribution in the middle of the genome: Test different midpoints for your rearranged distribution to be around; this could be after you already have a rearrangement method that works when we know the midpoint is in the middle. Write a method that uses that rearrangement method multiple times, with different midpoints, to find the best one, and therefore estimate the mutant position.
 - This isn't neccesary if the GA will work with a theoretical distribution in the qq plot

